lib/index.js

```diff
@@ -407,7 +407,7 @@ client.on('message', message => {
                serverId, args[0]
              ).then(data => {
                serversConfigStore = []
-               if (data) { message.channel.send("Prefix changed."); } if (data) client.guilds.cache.forEach(server => {
+               if (data) message.channel.send("Prefix changed."); client.guilds.cache.forEach(server => {
```

Comment:
This invalidates if statement for guild cache function, could you update it so it looks like this? 
![image](https://user-images.githubusercontent.com/11031051/102268094-843e1b00-3f23-11eb-8d74-ccead7782f66.png)